plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -103,7 +103,7 @@ public static void main(String[] args)
     {
         Logging.initialize();
 
-        TestingSqlServer testingSqlServer = new TestingSqlServer();
+        TestingSqlServer testingSqlServer = new TestingSqlServer(true);
```

Comment:
I do not get the logic where you choose `true` and where `false`.
I would use the value which is more common in the real world for all test + add on smoke test to test with opposite value.

Reply:
I want to run both - with the RCSI enabled (preferred if available as it has stronger guarantees on reads) and disabled so there is no other explanation to that.

Reply:
@losipiuk are you concerned about _new_ usages of `TestingSqlServer` class?
how do i choose the right value for a new test? i had a concern like that too, but didn't write it.

Reply:
My concern was that I did not understand why some classes use `true` and some `false`. I know that we want to test both, but I did not understand specificly why choices were made to test *this* test with `true` and *that other* test with `false`.
E.g we are using `false` in `TestSqlServerIntegrationSmokeTest` and `true` in `TestSqlServerDistributedQueries`. Why? I am not saying it is wrong, just I do not understand the choice.

If I was doing that I would probably set it to `true` for all the tests we had so far.
And add yet another test class (totally new one or extension of TestSqlServerIntegrationSmokeTest), and test it with `false`.

Reply:
@losipiuk I did that and reverted due to @findepi concern on tests execution times. I'm confused...

Reply:
Let's chat

Reply:
Ok. So let's keep snapshot isolation enabled as default and a test that runs without that.